src/libaktualizr/uptane/tuf.cc

```diff
@@ -69,6 +69,53 @@ std::ostream &Uptane::operator<<(std::ostream &os, const Hash &h) {
   return os;
 }
 
+std::string Hash::encodeVector(const std::vector<Uptane::Hash> &hashes) {
+  std::stringstream hs;
+
+  for (auto it = hashes.cbegin(); it != hashes.cend(); it++) {
+    hs << it->TypeString() << ":" << it->HashString();
+    if (std::next(it) != hashes.cend()) {
+      hs << ";";
+    }
+  }
+
+  return hs.str();
+}
+
+std::vector<Uptane::Hash> Hash::decodeVector(std::string hashes_str) {
+  std::vector<Uptane::Hash> hash_v;
+
+  std::string cs = std::move(hashes_str);
```

Comment:
Why bother with a new variables and a move?

Reply:
`cs` is modified afterwards. I could modify the argument directly but I prefer the style where mutable variables are declared separately.

Reply:
That's fine, I figured it was just a style choice but just wanted to be sure!